image_classification.py

```diff
@@ -0,0 +1,108 @@
+# Training a custom image classification model with the trainer API
+# docs : https://huggingface.co/docs/transformers/en/main_classes/trainer
+# adapted from : https://colab.research.google.com/drive/1u9r-p_x7QXH9zAbQ5c0O2smEBHvC44me?usp=sharing
+import evaluate
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from datasets import Image, load_dataset
+from torchvision import transforms
+from transformers import Trainer, TrainingArguments
+
+# Constants
+accuracy = evaluate.load("accuracy")
+dataset = load_dataset("mnist")
+
+# Processing the dataset
+dataset = dataset.cast_column("image", Image(mode="RGB"))
+transform = transforms.Compose([transforms.ToTensor()])
+
+
+def to_pt(batch):
+    batch["pixel_values"] = [transform(image.convert("RGB")) for image in batch["image"]]
+    return batch
```

Comment:
Let's make this not a 4-leter function and declare what it's doing with comments please :) (Why do we need to convert to PyTorch? What's happening as a result?)